server/websocket.go

```diff
@@ -465,83 +465,23 @@ func (p *Plugin) handleLeave(us *session, userID, connID, channelID string) erro
 	return nil
 }
 
-func (p *Plugin) handleJoin(userID, connID, channelID, title, threadID string) error {
-	p.LogDebug("handleJoin", "userID", userID, "connID", connID, "channelID", channelID)
+func (p *Plugin) handleJoin(userID, connID string, req CallStartRequest) error {
```

Comment:
It might be a little confusing to have `CallStartRequest` as a param to `handleJoin` -- even if the data is the same. What do you think?